open_dread_rando/door_patcher.py

```diff
@@ -295,14 +306,16 @@ def set_door_type(self, door: Container, door_type: DoorType, scenario: str):
             door.pComponents.LIFE.bStayOpen = False
 
     def create_shield(self, scenario: str, door: Container, shield_data: ActorData, dir: str):
-        # make shields
-        shield = self.editor.copy_actor(scenario, door.vPos, self.SHIELD, f"{door.sName}{dir}")
+        # make a new RandoShield by popping the lowest actor
+        shield = self.editor.copy_actor(scenario, door.vPos, self.SHIELD, f"RandoShield_{self.available_shield_ids[scenario].pop(0)}")
```

Comment:
The code for making this id shows up twice. You should make a function that does it